os/board/rtl8730e/src/component/wifi/inic/inic_ipc_host_api.c

```diff
@@ -323,13 +333,37 @@ static void inic_ipc_print_int_hdl(VOID *Data, u32 IrqStatus, u32 ChanNum)
 	(void) IrqStatus;
 	(void) ChanNum;
 
+	static km4log_msg_t message_event = { 0 };
+
+	/* receive a log message over IPC */
 	PIPC_MSG_STRUCT ipc_recv_msg = (PIPC_MSG_STRUCT)ipc_get_message(IPC_NP_TO_AP, IPC_N2A_NP_LOG_CHN);
 	char *tmp_buffer = (char *)ipc_recv_msg->msg;
 	DCache_Invalidate((u32)tmp_buffer, ipc_recv_msg->msg_len);
-	
-	/* Print out buffer */
-	lldbg_noarg("%s",tmp_buffer);
 
+	/* pick a buffer to fill. for rapid processing, assume that the higher priority task has already consumed the buffer, otherwise overwrite it */
+	u8 buf_idx = g_inic_ipc_logging_buf_ctr;
```

Comment:
Is this line necessary? why not just use g_inic_ipc_logging_buf_ctr for all buf_idx below